packages/paper/src/paper.ts

```diff
@@ -36,9 +37,9 @@ import {
 // > https://exploringjs.com/deep-js/ch_proxies.html
 setAutoFreeze(false);
 
-export class Paper<UserOperations extends OperationMap = OperationMap> {
-  protected history: DocumentStore[] = [];
-  protected current: DocumentStore;
+export class Paper<ST extends SchemaTypes = SchemaTypes, UserOperations extends OperationMap = OperationMap> {
```

Comment:
This is considered a breaking change as the first type argument is no longer the `UserOperations`.